cypress/utils/helpers.js

```diff
@@ -10,3 +10,11 @@ export function pickTargetProducts(category, quantity = 1, isShopPage = false) {
     : _.values(category.items);
   return _.sampleSize(items, quantity);
 }
+
+export function getSubString(givenString) {
+  return givenString.substr(_.random(0, givenString.length), _.random(0, givenString.length));
+}
+
+export function generateRandomSearchQuery() {
+  return Math.random().toString(36).substring(2, 7);
```

Comment:
what does `.toString(36)` mean?

Reply:
I'm not sure you need this generator - it is not reusable
just hard-code your search query to the test

Reply:
just make it more simple to understand

Reply:
done